ntbs-service/wwwroot/source/app.ts

```diff
@@ -33,4 +33,7 @@ if (config.env === 'development') {
   module.hot.accept()
 }
 
-govUkJsInitAll()
\ No newline at end of file
+govUkJsInitAll();
+
+// For compatibility with IE11
+require('es6-promise').polyfill();
```

Comment:
Do you think we should check which browser we're on first before applying this polyfill? It still works on Chrome of course, but obviously the normal ES6 syntax worked too..
It doesn't seem to be super straightforward to detect IE as the browser from what I saw.

Reply:
(well, it's straightforward, but not pretty ha)

Reply:
If we want to do that, then we should get polyfill in a script directly in html, rather than in `app.ts` - everything in `app.ts` gets bundled up, so any conditional logic would just get bundled too! We could do 2 bundles, but that feels like overkill! Do you know how much this pollyfil is adding to the bundle size?

Also, I imagine `es6-promise` is an npm library? If so, you're missing the `package.json` and `lock` file changes in this commit!